plugins/@grouparoo/dbt/src/settings/parse.ts

```diff
@@ -122,8 +122,8 @@ const parseProject: ParseProjectMethod = async (args) => {
     );
   }
 
-  const contents = fs.readFileSync(projectPath);
-  const document = yaml.load(contents);
+  const contents = String(fs.readFileSync(projectPath));
```

Comment:
I feel like we've discussed this, but I believe this should be:
```suggestion
  const contents = fs.readFileSync(projectPath, {encoding: "utf8"});
```

Reply:
I don't think this is a great idea - you can't assume a user's hard drive is in a certain encoding

Reply:
I thought this set the expectation for what _we_ want to work with. Docs don't explicitly say no, but imply that's what we're reading. Okay!